docs-source/content/userguide/managing-fmw-domains/fmw-infra/_index.md

```diff
@@ -124,10 +125,10 @@ cd docker-images/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s
 
 This will produce an image named `oracle/fmw-infrastructure:12213-update-k8s`.
 
-All samples and instructions reference the pre-built and already patched image, `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3`. When building your own image, you will need to rename `oracle/fmw-infrastructure:12213-update-k8s` to `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3`.
+All samples and instructions reference the pre-built image, `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.4`. When building your own image, you will need to rename `oracle/fmw-infrastructure:12213-update-k8s` to `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.4`.
```

Comment:
When building your own image, you will need to rename `oracle/fmw-infrastructure:12213-update-k8s` to `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.4`. 
Should this be: When building your own image, you will need to rename `oracle/fmw-infrastructure:12214-update-k8s` to `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.4`.

Reply:
@mriccell, the docker-images project doesn't have samples for 12.2.1.4.  I'll try to reword around this now, but we should fix this. :)